appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -4920,7 +4777,8 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
 
   @DefaultMessage("<p><b>We detected errors while reading in your project</b></p>" +
                 "<p>To protect your project from damage, we have ended this session. You may close this " +
-                "window.</p>")
+                "window.</p>" +
+                "<p>For more information see <a href=\"http://appinventor.mit.edu/explore/ai2/support/troubleshooting.html#Corrupt\" target=\"_blank\">the Trouble Shooting Page</a></p>")
```

Comment:
I thought this used to be in the branding branch. Just want to check that moving it here is intentional.

Reply:
Nope, not intentional and removed.